draft-ietf-tls-wkech.xml

```diff
@@ -693,12 +727,12 @@ Client <----------->|                    | facing
         </t>
 	<t>
 	    Operators should be careful to avoid directory enumeration or traversal attacks
-	    if the .well-known/ URLs are served directly from a filesystem.  For example, a
+	    if the well-known URLs are served directly from a filesystem.  For example, a
 	    misconfiguration of this kind could expose the ECH private keys.
 	</t>
 
         <t>
-            As described, in mutlti-CDN and simlar scenarios, a ZF might
+            As described, in multi-CDN and simlar scenarios, a ZF might
```

Comment:
s/described/described earlier/ ??

Reply:
hmm, we don't actually seem to describe that earlier (well, not fully), so maybe s/As described, in multi-CDN/In multi-CDN/